DynamicSmoothedGUI.py

```diff
@@ -366,9 +366,9 @@ def getLocation(text: str) -> (int, int):
     firstNum = firstNum * 100
     secondNum = secondNum * 100
     firstNum = firstNum + tile_num_width * tile_size / 2
-    secondNum = secondNum + tile_num_height * tile_size / 2
+    secondNum = -secondNum + tile_num_height * tile_size / 2
 
-    return (firstNum, -secondNum)
+    return (firstNum, secondNum)
```

Comment:
The axis were accidentally reversed. This meant that when a user entered a coordinate for the robot to go, it would not go to the right quadrant. These negations fix that!